utility/src/test/java/org/oppia/android/util/profile/ProfileNameValidatorTest.kt

```diff
@@ -0,0 +1,71 @@
+package org.oppia.android.util.profile
+
+import com.google.common.truth.Truth.assertThat
+import org.junit.Before
+import org.junit.Test
+import javax.inject.Inject
+
+class ProfileNameValidatorTest {
+  @Inject
+  lateinit var profileNameValidator: ProfileNameValidator
+
+  @Before
+  fun setup() {
+    profileNameValidator = ProfileNameValidator()
+  }
+
+  @Test
+  fun testIsNameValid_nameWithSpaces_returnsFalse() {
+    val nameWithSpaces = "Ben Henning"
+    assertThat(profileNameValidator.isNameValid(nameWithSpaces)).isFalse()
+  }
+
+  @Test
+  fun testIsNameValid_nameWithNumber_returnsFalse() {
+    val nameWithNumber = "Jishnu7"
+    assertThat(profileNameValidator.isNameValid(nameWithNumber)).isFalse()
+  }
+
+  @Test
+  fun testIsNameValid_nameWithDisallowedSymbol_returnsFalse() {
```

Comment:
Since we were waiting for parameterized tests, I suggest using that here, instead, to avoid the for-loop/control flow below. See https://github.com/oppia/oppia-android/blob/develop/utility/src/test/java/org/oppia/android/util/math/MathExpressionParserTest.kt for an example test that uses parameterization.

Ditto for other tests that are verifying multiple things.

Reply:
done :)